### PR TITLE
fix: argument `--accent` behaves like the old argument

### PR DESCRIPTION
**Pull request by isabelroses**
tested with the following commands
- `python3 build.py mocha -a rosewater all --dest ./dist`
- `python3 build.py mocha -a rosewater sapphire --dest ./dist`
- `python3 build.py mocha -a sapphire --dest ./dist`

### CODE REVIEW

File: build.py [hunk 1]
```diff
@@ -591,50 +592,29 @@ def main():
...
-            )
 
-            tweaks_temp()
-            gnome_shell_version()
-            build_theme(ctx)
-            logger.info("Done!")
-    else:
-        accent = getattr(palette.colors, args.accent)
+    accents = args.accents
+    if "all" in accents:
```
Comment (nullishamy) [inline build.py:597]: Hmm. Not sure how much I like the special value being kept around. Is there a better way to do this? Perhaps a flag like `--all-accents`?

File: build.py [hunk 2]
```diff
@@ -660,7 +640,6 @@ def main():
         build_theme(ctx)
         logger.info("Done!")
```
Comment (nullishamy) [inline build.py:641]: Ideally this log would be moved out and only printed once we're actually done. Maybe replace it with an empty log so we can more easily separate each build in the logs?

**Comment by nullishamy** [comment]
Do we want to fix CI here as well?

**Comment by isabelroses** [comment]
> Do we want to fix CI here as well?

good catch

**Comment by nullishamy** [comment]
Related, and for another PR, maybe we want to just run CI on every PR? Build only of course.

**Review by nullishamy — APPROVED** [review]

**Pull request merged**